src/Serilog.Sinks.AzureDataExplorer/Serilog.Sinks.AzureDataExplorer.csproj

```diff
@@ -22,7 +22,6 @@ Improved options default values
     </PackageReleaseNotes>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
```

Comment:
Apache or MIT ? If so we need to add MIT license and license file for that

Reply:
Agreed, should have been MIT. did the change.